src/ext/conversion/exportProperties.ts

```diff
@@ -1,12 +1,36 @@
 import conversion from 'qlik-object-conversion';
+import { setValue } from 'qlik-chart-modules';
 
-import { ExportFormat, PropTree } from '../../types';
+import { DimensionProperties, ExportFormat, MeasureProperties, PropTree } from '../../types';
 
-const exportProperties = (propertyTree: PropTree, hyperCubePath: string): ExportFormat => {
+export const getColumnWidth = (colIdx: number, qDimensions: DimensionProperties[], qMeasures: MeasureProperties[]) => {
+  const numDims = qDimensions.length;
+  const isDim = colIdx < numDims;
+  const info = isDim ? qDimensions[colIdx] : qMeasures[colIdx - numDims];
+
+  const { columnWidth } = info.qDef;
+
+  // For converting "sn-table to table", columnWidth is -1 unless the width type is pixels.
+  return columnWidth?.type === 'pixels' ? columnWidth.pixels : -1;
+};
+
+const exportProperties = (propertyTree: PropTree, hyperCubePath?: string): ExportFormat => {
   const expFormat = conversion.hypercube.exportProperties({
     propertyTree,
     hyperCubePath,
   });
+  const {
+    qHyperCubeDef: { qColumnOrder, qDimensions, qMeasures },
+  } = expFormat.properties;
+  const columnsLength = qDimensions.length + qMeasures.length;
+  const columnOrder = qColumnOrder?.length === columnsLength ? qColumnOrder : Array.from(Array(columnsLength).keys());
+  const columnWidths = columnOrder.map((colIdx: number) => getColumnWidth(colIdx, qDimensions, qMeasures));
```

Comment:
you could bake this into one function, then it would be easy to test if it works without `qColumnOrder`.

Reply:
Fixed